nixos/tests/fcitx/config

```diff
@@ -0,0 +1,147 @@
+[Hotkey]
+# Trigger Input Method
+#TriggerKey=CTRL_SPACE
```

Comment:
Do we need all these commented out settings?

Reply:
I cleared them up now.